### ISSUE TITLE
Peer Dependency React

### ISSUE THREAD

**Issue by Zeratoxx** [cymen/react-floating-label-input#34]
Hi! I am using react 17, but here, the peer deps are react 15^ or 16^, so I am getting this error when running `npm install`:  

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: laser-chess@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-floating-label-input@4.3.2
npm ERR! node_modules/react-floating-label-input
npm ERR!   react-floating-label-input@"^4.3.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by Zeratoxx** [comment]
Running `npm install --legacy-peer-deps` goes on with the installation, but I don't know if everything is working properly

**Comment by cymen** [comment]
Oh whoops, I'll fix this, sorry.

**Comment by cymen** [comment]
@Zeratoxx This is fixed with version 4.3.3 that I just published. My apologies again for missing this -- didn't see it until today.

**Comment by Zeratoxx** [comment]
It's okay, thank you for your response anyway :)
